seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java

```diff
@@ -90,28 +114,34 @@ public void deserialize(SourceRecord record, Collector<SeaTunnelRow> collector)
         Schema valueSchema = record.valueSchema();
 
         Struct sourceStruct = messageStruct.getStruct(Envelope.FieldName.SOURCE);
-        // TODO: multi-table
+        String database = sourceStruct.getString(DATABASE_NAME_KEY);
         String tableName = sourceStruct.getString(AbstractSourceInfo.TABLE_NAME_KEY);
+        String tableId = database + ":" + tableName;
+        SeaTunnelRowDebeziumDeserializationConverters converters = multipleRowConverters.getOrDefault(tableId, singleRowConverter);
```

Comment:
The judgment on the single table is missing

Reply:
through `multipleRowConverters.getOrDefault(..., singleRowConverter)`